### ISSUE TITLE
403 Response presented as "Resize not allowed"

### ISSUE THREAD

**Issue by eastlondoner** [pkgcloud/pkgcloud#405]
pkgcloud\lib\pkgcloud\openstack\client.js

Client.prototype.failCodes = {
  400: 'Bad Request',
  401: 'Unauthorized',
  403: 'Resize not allowed',
...
};

This is confusing when you receive a 403 error e.g. trying to remove a Virtual Interface from a host as "Resize not Allowed" doesnt make any sense, clearly all 403 errors are not "Resize not allowed". This should read "Forbidden" or  maybe just "Not allowed".

**Comment by kenperkins** [comment]
Agreed that `403: Resize not allowed` doesn't make sense. This probably would be a breaking change, since it's possible some users would check on text and not the code. @indexzero @rosskukulinski
